qutip/tests/solver/test_countstat.py

```diff
@@ -35,16 +36,26 @@ def test_dqd_current():
         H = (eps/2 * sz + tc * sx)
         L = qutip.liouvillian(H, c_ops)
         rhoss = qutip.steadystate(L)
-        current[n], noise[n] = qutip.countstat_current_noise(L, [],
-                                                             rhoss=rhoss,
-                                                             J_ops=J_ops)
+        current_, noise_ = qutip.countstat_current_noise(
+            L, [], wlist=[0,1], rhoss=rhoss, J_ops=J_ops
+        )
+        current[n] = current_[0]
+        noise[n] = noise_[0,0,0]
 
         current2 = qutip.countstat_current(L, rhoss=rhoss, J_ops=J_ops)
         assert abs(current[n] - current2) < 1e-8
 
         current2 = qutip.countstat_current(L, c_ops, J_ops=J_ops)
         assert abs(current[n] - current2) < 1e-8
 
+        current2 = qutip.countstat_current(L, c_ops)[0]
+        assert abs(current[n] - current2) < 1e-8
+
+        current3, noise3 = qutip.countstat_current_noise(L, c_ops, wlist=[0,1],
+                                                         sparse=False)
+        assert abs(current_[0] - current3[0]) < 1e-6
+        np.testing.assert_allclose(noise_[0, 0, :], noise3[0, 0, :], atol=1e-6)
```

Comment:
Could we sort out the variable names and types a bit? It looks we're comparing numpy arrays, e.g. `current2` and scalars, `e.g. current[n]`, and we have `current_`, but `current3` and `current2`. Maybe just `current_1`, `current_2` and `current_3` and always have each be the same type of object?